src/bson.h

```diff
@@ -48,7 +48,7 @@ const int preLoadedIndex = 10000;
 #define NanToString(obj) (Unmaybe(obj->ToString(Nan::GetCurrentContext())))
 #define NanToObject(obj) (Unmaybe(obj->ToObject(Nan::GetCurrentContext())))
 
-#if NODE_MAJOR_VERSION >= 10
+#if NODE_MAJOR_VERSION >= 12
```

Comment:
what required these changes?

Reply:
The prebuild for electron 3.0.0 was failing with a bunch of errors that seemed related to the updates made in #51 - I tried bumping the conditional up to 12 since that PR was for Node 12 compatibility, and now all the prebuilds work.